CHANGELOG.md

```diff
@@ -1,3 +1,310 @@
+## 3.0.0 ( Wed Jun 20 2018 11:06:21 GMT+0200 (CEST) )
+
+
+## Breaking changes
```

Comment:
Breaking changes : 

- Drop support for node 0.12
- Drop gracefulReload command
- Remove Interactor from PM2 source code
- Replace pmx by @pm2/io

Reply:
Done